pkg/logutil/redact.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 PingCAP, Inc. Licensed under Apache-2.0.
+
+package logutil
+
+import (
+	"fmt"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var enableRedactLog bool
+
+// InitRedact inits the enableRedactLog
+func InitRedact(redact bool) {
+	enableRedactLog = redact
+}
+
+// NeedRedact returns whether to redact log
+func NeedRedact() bool {
+	return enableRedactLog
+}
```

Comment:
please use `pingcap/errors`'s [RedactLogEnabled](https://pkg.go.dev/github.com/pingcap/errors@v0.11.5-0.20201126102027-b0a155152ca3#pkg-variables) variable instead, which simplifies the handling for BR-in-SQL.